spec/std/isa/inst_subtype/I/I-x-x.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2025 Usman Akinyemi
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../schemas/inst_subtype_schema.json
+
+$schema: inst_subtype_schema.json#
+kind: instruction_subtype
+name: I-x-x
+
+data:
+  type: { "$ref": "inst_type/I.yaml#" }
+  subtype: { "$ref": "inst_subtype/I/I-x-x.yaml#" }
+  opcodes:
+    $inherits: inst_type/I.yaml#/opcodes
+  variables:
+    func11:
```

Comment:
Sorry, I should've caught this before now. This is a 12-bit field, so "func11" is confusing. However, in the spec, it's pretty consistently referred to as "imm", so let's use that name here.